qcfractal/cli/qcfractal_manager.py

```diff
@@ -74,47 +66,54 @@ def main(args=None):
-            queue_client = dd.Client(args["dask_uri"])
-
-    # Handle Fireworks adapters
-    elif args["adapter_type"] == "fireworks":
-
-        # Check option conflicts
-        num_options = sum(args[x] is not None for x in ["fw_config", "fw_uri"])
-        if num_options == 0:
-            args["fw_uri"] = "mongodb://localhost:27017"
-        elif num_options != 1:
-            raise KeyError("Can only provide a single URI or config_file for Fireworks.")
-
-        fireworks = cli_utils.import_module("fireworks")
-
-        if args["fw_uri"] is not None:
-            queue_client = fireworks.LaunchPad(args["fw_uri"], name=args["fw_name"])
-        elif args["fw_config"] is not None:
-            queue_client = fireworks.LaunchPad.from_file(args["fw_config"])
-        else:
-            raise KeyError("A URI or config_file must be specified.")
-
-    elif args["adapter_type"] == "executor":
+    # if args["adapter_type"] == "dask":
+    #     dd = cli_utils.import_module("distributed")
+
+    #     if args["local_cluster"]:
+    #         # Build localcluster and exit callbacks
+    #         queue_client = dd.Client(threads_per_worker=1, n_workers=args["local_workers"])
+    #     else:
+    #         if args["dask_uri"] is None:
+    #             raise KeyError("A 'dask-uri' must be specified.")
+    #         queue_client = dd.Client(args["dask_uri"])
+
+    # # Handle Fireworks adapters
+    # elif args["adapter_type"] == "fireworks":
+
+    #     # Check option conflicts
+    #     num_options = sum(args[x] is not None for x in ["fw_config", "fw_uri"])
+    #     if num_options == 0:
+    #         args["fw_uri"] = "mongodb://localhost:27017"
+    #     elif num_options != 1:
+    #         raise KeyError("Can only provide a single URI or config_file for Fireworks.")
+
+    #     fireworks = cli_utils.import_module("fireworks")
+
+    #     if args["fw_uri"] is not None:
+    #         queue_client = fireworks.LaunchPad(args["fw_uri"], name=args["fw_name"])
+    #     elif args["fw_config"] is not None:
+    #         queue_client = fireworks.LaunchPad.from_file(args["fw_config"])
+    #     else:
+    #         raise KeyError("A URI or config_file must be specified.")
+
+    if args["cores"] is None:
+        args["cores"] = qcng.config.get_global("ncores")
+
+    if args["memory"] is None:
+        args["memory"] = qcng.config.get_global("memory")
+
+    if args["adapter_type"] == "executor":
 
         from concurrent.futures import ProcessPoolExecutor
 
-        queue_client = ProcessPoolExecutor(max_workers=args["nprocs"])
+        queue_client = ProcessPoolExecutor(max_workers=args["ntasks"])
 
     else:
-        raise KeyError("Unknown adapter type '{}', available options: 'fireworks', 'dask', .".format(
-            args["adapter_type"]))
+        raise KeyError(
+            "Unknown adapter type '{}', available options: 'fireworks', 'dask', .".format(args["adapter_type"]))
```

Comment:
Probably should change this since you disabled fireworks and dask for now

Reply:
Good point. Lets ear mark to delete later.